sql-join/where-are-they.sql

```diff
@@ -0,0 +1,5 @@
+select "addresses"."line1",
+       "cities"."name",
+       "addresses"."district"
+from "addresses"
+join "cities" using ("cityId");
```

Comment:
you are missing the country names good sir!

Reply:
![Screen Shot 2021-01-12 at 3 17 22 PM](https://user-images.githubusercontent.com/69181727/104386192-7e6b3380-54e9-11eb-9fe1-74fc5eedb3dc.png)

Reply:
Updated 👍